test/RibbonVolatility.js

```diff
@@ -660,6 +664,89 @@ function behavesLikeRibbonVolatility(params) {
         assert.equal(await this.contract.numOfPositions(user), 1);
       });
     });
+
+     //REMOVE .skip to run
+    describe.skip("#claimRewards", () => {
+      let snapshotId;
+      let rhegicContract;
+      let withSinger;
+      let prov = ethers.getDefaultProvider();
+
+      beforeEach(async function () {
+        snapshotId = await time.takeSnapshot();
+        rhegicContract = new ethers.Contract(rHEGICJSON.address, rHEGICJSON.abi, prov);
+        withSinger = await rhegicContract.connect(await ethers.provider.getSigner(user));
+      });
+
+      afterEach(async () => {
+        await time.revertToSnapShot(snapshotId);
+      });
+
+      it("can claim valid ETH rewards", async function () {
+        let balanceBefore1 = await withSinger.balanceOf("0x390df0394ef2930eae1e3a610202d644fc21127c");
+        let balanceBefore2 = await withSinger.balanceOf("0x0c1f3df3524c4a5c4d9e1471003800f9595a05d0");
+        const res = await this.contract.claimRewards(
+                      HEGIC_PROTOCOL,
+                      HEGIC_ETH_REWARDS,
+                      [BigNumber.from("2975"), BigNumber.from("2974")]
+                    )
+        const receipt = await res.wait();
+        let balanceAfter1 = await withSinger.balanceOf("0x390df0394ef2930eae1e3a610202d644fc21127c");
+        let balanceAfter2 = await withSinger.balanceOf("0x0c1f3df3524c4a5c4d9e1471003800f9595a05d0");
+        assert.isAtLeast(balanceAfter1 - balanceBefore1, 1000000000000000000);
+        assert.isAtLeast(balanceAfter2 - balanceBefore2, 1000000000000000000);
+      });
+
+      it("can claim valid WBTC rewards", async function () {
+        let balanceBefore1 = await withSinger.balanceOf("0x37f32c51882e2b7f5195d948cc386d925892d49b");
+        let balanceBefore2 = await withSinger.balanceOf("0x6b0bde6174bf2f10ed64bef05ec1df4ee4f1187b");
+        const res = await this.contract.claimRewards(
+                      HEGIC_PROTOCOL,
+                      HEGIC_WBTC_REWARDS,
+                      [BigNumber.from("1399"), BigNumber.from("1400")]
+                    )
+        const receipt = await res.wait();
+        let balanceAfter1 = await withSinger.balanceOf("0x37f32c51882e2b7f5195d948cc386d925892d49b");
+        let balanceAfter2 = await withSinger.balanceOf("0x6b0bde6174bf2f10ed64bef05ec1df4ee4f1187b");
+        assert.isAtLeast(balanceAfter1 - balanceBefore1, 1000000000000000000);
+        assert.isAtLeast(balanceAfter2 - balanceBefore2, 1000000000000000000);
+      });
+
+      // Something with hardhat cant recognize reversion expect: it will show as error but you will see
+      // revert error matches what is expected
+
+      it("reverts on invalid ETH optionid", async function () {
+        await expect(await this.contract.claimRewards(
```

Comment:
Fix the revert by removing the `await`:
```
await expect(this.contract.claimRewards(
```

This is because the promise was rejected and was thrown before the `expect` function was called. Applies to all the revert tests below.